internal/tfengine/tfengine_test.go

```diff
@@ -54,7 +52,10 @@ func TestExamples(t *testing.T) {
 			path := filepath.Join(tmp, "live")
 
 			// Convert the configs not reference a GCS backend as the state bucket does not exist.
-			convertToLocalBackend(t, path)
+			err = ConvertToLocalBackend(path)
+			if err != nil {
```

Comment:
if err := ConvertToLocalBackend(); er != nil

also log func name and path in the t.Fatal log